### PR TITLE
implementing calculator ui

### PR DESCRIPTION
**Pull request by DiegoDiazGlobant**
Implementation of a calculator layout design

Nexus S - preview
screen_nexus

Pixel 3a - preview
screen_pixel

App icon
screen_icon

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -8,14 +8,18 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.Calculator">
-        <activity android:name=".MainActivity">
+        android:theme="@style/Theme.AppCompat.NoActionBar">
+        <activity android:name=".MainActivity"        android:screenOrientation="portrait">
```
Comment (mauroLonghi) [inline app/src/main/AndroidManifest.xml]: please put a enter

Comment (benjaglobant) [inline app/src/main/AndroidManifest.xml]: Please put android:screenOrientation in a line down of android:name

File: app/src/main/res/values/font_certs.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        </item>
+    </string-array>
+</resources>
```
Comment (mauroLonghi) [inline app/src/main/res/values/font_certs.xml:11]: it you use this resource?

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="tittle_style">
```
Comment (benjaglobant) [inline app/src/main/res/values/styles.xml]: Refactor to "title_style" the name

**Review by BQuinteros — APPROVED** [review]

File: app/src/main/res/values/dimens.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    <dimen name="activity_main_margin_tittle">20dp</dimen>
+    <dimen name="activity_main_margin_icon">20dp</dimen>
+    <dimen name="activity_main_size_icon">45dp</dimen>
+    <dimen name="activity_main_padding_top_screen">50dp</dimen>
+    <dimen name="activity_main_padding_end_screen">15dp</dimen>
+    <dimen name="activity_main_text_size_screen_operation">30sp</dimen>
+    <dimen name="activity_main_text_size_screen_result">40sp</dimen>
+    <dimen name="activity_main_padding_bottom_screen_result">10dp</dimen>
+    <dimen name="activity_main_margin_bottom_screen_result">40dp</dimen>
+    <dimen name="activity_main_height_button">0dp</dimen>
```
Comment (SartiMau) [inline app/src/main/res/values/dimens.xml]: the 0dp when it refers to a height or width should be in the XML, without a dimen res

**Review by SartiMau — CHANGES_REQUESTED** [review]
Pls copnvert all png files that you add to webp

File: app/src/main/res/font/carter_one.xml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<font-family xmlns:app="http://schemas.android.com/apk/res-auto"
```
Comment (LuchoElissondoGlobant) [inline app/src/main/res/font/carter_one.xml:2]: nice!

Comment (lucianoeli) [inline app/src/main/res/font/carter_one.xml:2]: Nice!

**Review by lucianoeli — APPROVED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (DiegoDiazGlobant) [reply app/src/main/AndroidManifest.xml]: solved!

Comment (DiegoDiazGlobant) [reply app/src/main/AndroidManifest.xml]: solved!

File: app/src/main/res/values/font_certs.xml (hunk 2 above)
Comment (DiegoDiazGlobant) [reply app/src/main/res/values/font_certs.xml:11]: No! Deleted..

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (DiegoDiazGlobant) [reply app/src/main/res/values/styles.xml]: Ups. changed

File: app/src/main/res/values/dimens.xml (hunk 4 above)
Comment (DiegoDiazGlobant) [reply app/src/main/res/values/dimens.xml]: Solved!

File: app/src/main/res/font/carter_one.xml (hunk 5 above)
Comment (DiegoDiazGlobant) [reply app/src/main/res/font/carter_one.xml:2]: Thanks!!

**Review by lucianoeli — APPROVED** [review]

**Review by benjaglobant — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**